documentation/source/users/cantherm/input.rst

```diff
@@ -135,7 +136,9 @@ Allowed atom symbols for the ``atoms`` parameter are
 
 Allowed bond types for the ``bonds`` parameter are, e.g., ``'C-H'``, ``'C-C'``, ``'C=C'``, ``'N-O'``, ``'C=S'``, ``'O=O'``, ``'C#N'``...
 
-(The element order is according to the order specified under "allowed atom symbols" above, i.e., ``'C=N'`` is correct while ``'N=C'`` is incorrect. Use ``-``/``=``/``#`` to denote a single/double/triple bond, respectively). For example, for formaldehyde we would write::
+``'O=S=O'`` is also allowed.
+
+The order of elements in for the bond correction is important and generally follows the order specified under "allowed atom symbols" above, i.e., ``'C=N'`` is correct while ``'N=C'`` is incorrect. Use ``-``/``=``/``#`` to denote a single/double/triple bond, respectively. For example, for formaldehyde we would write::
```

Comment:
related to the comment i made before, maybe add atom order list here, [C, N, O, S, H...]